scipy/stats/_discrete_distns.py

```diff
@@ -514,6 +526,142 @@ def _logcdf(self, k, M, n, N):
 hypergeom = hypergeom_gen(name='hypergeom')
 
 
+class nhypergeom_gen(rv_discrete):
+    r"""A negative hypergeometric discrete random variable.
+
+    Consider a box containing :math:`N` balls:, :math:`K` red and
+    :math:`N-K` blue. We randomly sample balls from the box, one
+    at a time and *without* replacement, until we have picked :math:`r`
+    blue balls. `nhypergeom` is the distribution of the number of
+    red balls :math:`k` we have picked.
+
+    %(before_notes)s
+
+    Notes
+    -----
+    The symbols used to denote the shape parameters (`N`, `K`, and `r`) are not
+    universally accepted. See the Examples for a clarification of the
+    definitions used here.
+
+    The probability mass function is defined as,
+
+    .. math:: f(k; N, K, r) = \frac{{{k+r-1}\choose{k}}{{N-r-k}\choose{K-k}}}
+                                   {{N \choose K}}
+
+    for :math:`k \in [0, K]` and the binomial coefficient is:
+
+    .. math:: \binom{n}{k} \equiv \frac{n!}{k! (n - k)!}.
+
+    It is equivalent to observing :math:`k` successes in :math:`k+r-1`
+    samples with :math:`k+r`'th sample being a failure. The former
+    can be modelled as a hypergeometric distribution. The probability
+    of the latter is simply the number of failures remaining
+    :math:`N-K-(r-1)` divided by the size of the remaining population
+    :math:`N-(k+r-1)`. This relationship can be shown as:
+
+    .. math:: NHG(k;N,K,r) = HG(k;N,K,k+r-1)\frac{(N-K-(r-1))}{(N-(k+r-1))}
+
+    where :math:`NHG` is negative hypergeometric distribution and
+    :math:`HG` is the hypergeometric distribution.
```

Comment:
State explicitly that these are the probability mass functions.  In some contexts, such notation might be more commonly used as the CDF.
```suggestion
    where :math:`NHG` is probability mass function (PMF) of the negative hypergeometric
    distribution and :math:`HG` is the PMF of the hypergeometric distribution.
```

Reply:
done!